### PR TITLE
Type Declarations

### PR DESCRIPTION
**Pull request by kylecorbelli**
Beta npm version published to `1.3.0-beta.3`

### CODE REVIEW

**Comment by kylecorbelli** [comment]
@kylecorbelli please update the version to a non-beta version.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dollarshaveclub/react-passage",
-  "version": "1.2.0",
+  "version": "1.3.0-beta.3",
```
Comment (yowainwright) [inline package.json]: Does it still need the `beta.3`?

File: package.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
   "module": "dist/index.es.js",
   "jsnext:main": "dist/index.es.js",
   "unpkg": "dist/index.js",
+  "types": "types/index.d.ts",
```
Comment (yowainwright) [inline package.json]: 😍

File: package.json (hunk 1 above)
Comment (yowainwright) [reply package.json]: I wanna be bold here. Usually TS people will happily submit `types` update prs.

**Pull request merged**